buddyboss/buddyboss-platform
# Fixed - If someone shares a youtube video to the activity feed and th…

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2062  .

### How to test the changes in this Pull Request:

1. Go to the Activity feed
2. Post any youtube video
3. Post a comment and edit the Activity feed. Now youtube video should not appear in the comments. 

### Proof Screenshots or Video

https://www.loom.com/share/b37047a96bc24dc6b456f3de139d2e96


### Other information:
